test/integration-test.js

```diff
@@ -0,0 +1,72 @@
+let chai = require('chai');
+let PepperMint = require('../index')
+
+chai.should();
+
+try {
+  var config = require('./integration-config.json');
+} catch (e) {
+  // no config; don't run tests
+  process.exit()
```

Comment:
When running `npm test`, won't this prevent it from running other tests?

Reply:
You're right! I was hoping for a more elegant solution than what was in the other test with it all wrapped in a big conditional block but I suppose I'll just do it that way for now.